src/App.js

```diff
@@ -44,19 +44,19 @@ function App() {
     fetch(URL)
       .then((response) => response.json())
       .then(async (data) => {
-        // first result in spreadsheet is null.
-        // So when retrieving all results, remove first result
-        if (mode == null) {
-          data.shift();
-        }
         setFetchingData(false);
-
-        const [first, ...results] = data;
-        setData([first, ...results]);
+        // first result in spreadsheet is null.
+        data.shift();
```

Comment:
`data.shift` should be only be done if all results are returned; a better way of doing it would be removing this item specifically when it gets returned.

Reply:
Initially, I tried separating the first null item after filtering the data but ran into a compile error for never using the _ var which contains the null. 
`const [ _ , ...uniqueVenuesArray ] = data.filter(...`

Reply:
I'd just leave the `data.shift` line alone/unchanged. Otherwise, this filtering looks good I think